tools/update-authors-json

```diff
@@ -76,6 +76,7 @@ def run_production():
         contribs = []
         for user in json_data:
             author = user.get('author')
+            assert author is not None  # Each list entry should have an "author" dict
```

Comment:
Hmm. The existing code around this looks kind of dubious, and I'm not 100% sure what error cases are actually expected.

Just below this, we have a try/except when we try to get the fields within the `author` dict... but it's not checking for `KeyError`. It's checking for `AttributeError`. That wouldn't happen if `author` just doesn't have those fields, but it *would* happen if `author` itself is `None`.

So I think the edit here that would preserve the current error behavior is to instead of an assert, write
```
if author is None:
  print("Unable ..."...)
  continue
```
Then cut the `except` block. I'm still not 100% sure that this is a situation that actually happens, and not just someone being paranoid, but we'd be avoiding a regression if it is. What do you think of that solution?